src/Native/System.Native/pal_networking.cpp

```diff
@@ -6,15 +6,16 @@
 #include "pal_utilities.h"
 
 #include <arpa/inet.h>
-#include <assert.h>
-#include <functional>
 #include <netdb.h>
 #include <netinet/in.h>
 #include <netinet/tcp.h>
 #include <sys/socket.h>
+#include <sys/ioctl.h>
 #include <unistd.h>
-#include <vector>
+#include <alloca.h>
 #include <errno.h>
+#include <assert.h>
+#include <vector>
```

Comment:
I'd prefer to try and keep the included alphabetical, unless there are dependencies between headers